.github/workflows/pypi_publish.yml

```
@@ -1,9 +1,12 @@
-name: Python Packaging
+name: Packaging and Publishing
 
 on:
   push:
-
   workflow_dispatch:
+    inputs:
+      version:
+        description: "The version to publish (e.g., v6.7.0dev4)"
```

@ChristoGrab 

```suggestion
        description: "The version to publish (e.g., 6.7.0dev4)"
```

Since we are scraping "v" from the release tag, we may need to omit it here as well.